napari/plugins/hook_specifications.py

```diff
@@ -331,27 +326,21 @@ def napari_write_vectors(path: str, data: Any, meta: dict) -> Optional[str]:
 
 @napari_hook_specification(historic=True)
 def napari_experimental_provide_function_widget() -> Union[
```

Comment:
maybe we should take out the `_widget` part of the name now?

Reply:
Makes sense. Should we just go with `provide_function` or `provide_analysis_function`? I'm open to either.

Reply:
`provide_funtion` sounds better to me